src/Codeception/Configuration.php

```diff
@@ -421,7 +421,11 @@ public static function outputDir()
         if (!self::$logDir) {
             throw new ConfigurationException("Path for logs not specified. Please, set log path in global config");
         }
-        $dir = self::$dir . DIRECTORY_SEPARATOR . self::$logDir . DIRECTORY_SEPARATOR;
+
+        $dir = self::$logDir . DIRECTORY_SEPARATOR;
+        if(strcmp(self::$logDir[0], "/") !== 0){
```

Comment:
No. It would fallback to current behaviour and treat the path as relative.
I don't easily have access to windows machine but could spin up a VM and
look at a further PR to refine this.

On 11 March 2015 at 19:26, Anton Chernyavski <notifications@github.com>
wrote:

> In src/Codeception/Configuration.php
> <https://github.com/Codeception/Codeception/pull/1769#discussion_r26246456>
> :
>
> > @@ -421,7 +421,11 @@ public static function outputDir()
> >          if (!self::$logDir) {
> >              throw new ConfigurationException("Path for logs not specified. Please, set log path in global config");
> >          }
> > -        $dir = self::$dir . DIRECTORY_SEPARATOR . self::$logDir . DIRECTORY_SEPARATOR;
> > +
> > +        $dir = self::$logDir . DIRECTORY_SEPARATOR;
> > +        if(strcmp(self::$logDir[0], "/") !== 0){
>
> Would it work on Windows?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Codeception/Codeception/pull/1769/files#r26246456>.
>